tests/ra_aid/tools/test_execution.py

```
@@ -0,0 +1,218 @@
+    
+    with patch("ra_aid.tools.handle_user_defined_test_cmd_execution.run_shell_command") as mock_run_cmd, \
+         patch("ra_aid.tools.handle_user_defined_test_cmd_execution.logger") as mock_logger:
+        
+        # Simulate run_shell_command raising an exception
+        mock_run_cmd.side_effect = Exception("Command failed")
+        
+        result = execute_test_command(
+            config,
+            "original prompt",
+            0,
+            True
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)